### PR TITLE
Updated German Translation

### PR DESCRIPTION
**Pull request by bastianleicht**
Hi, I just updated the German Translation. Some Sentences didn't even make sense. Now they do :^)

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -173,17 +173,17 @@ msgstr "Schlüsseldatei:"
 #: src/Widgets/AccountEditor.vala:129
 #, fuzzy
```
Comment (ryonakano) [inline po/de.po:174]: Remove this line

**Review by ryonakano — CHANGES_REQUESTED** [review]
You need to remove these "fuzzy" lines, otherwise your translation changes will not be reflected in the app.

**Comment by bastianleicht** [comment]
Like that?

**Comment by ryonakano** [comment]
Yes that's right, thank you.

**Review by ryonakano — APPROVED** [review]

**Pull request merged**